taxi/forms.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+from taxi.models import Driver, Car
+from django.contrib.auth import get_user_model
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + (
+            "first_name",
+            "last_name",
+            "license_number",)
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+
+class CarForm(forms.ModelForm):
+    drivers = forms.ModelMultipleChoiceField(
+        queryset=get_user_model().objects.all(),
```

Comment:
The `queryset` for the `drivers` field in `CarForm` is currently set to `get_user_model().objects.all()`. This should be filtered to only include instances of `Driver`, as not all users may be drivers. You can achieve this by using `Driver.objects.all()` instead of `get_user_model().objects.all()`.